### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by sQVe** [neoclide/coc-stylelint#18]
Hey 👋,

I just made the switch to use `coc` for diagnostics instead of ALE but I cannot for the life of me get this extension to work. It doesn't show any errors or issues where it should and running `    CocCommand stylelint.executeAutofix` does nothing.

I think something is broken here after the last diagnostic tweaks of `coc.nvim`. 

Could I help out and provide any further info, @chemzqm?

**Comment by sQVe** [comment]
This is the output of `:CocInfo`
```
## versions

vim version: NVIM v0.5.0-dev+944-g82100a6bd
node version: v12.20.0
coc.nvim version: 0.0.79-5a4d0212bf
coc.nvim directory: /home/sqve/.local/share/nvim/plugged/coc.nvim
term: alacritty
platform: linux

## Log of coc.nvim

2020-12-18T14:01:26.937 INFO (pid:190427) [services] - registered service "languageserver.bash"
2020-12-18T14:01:26.951 INFO (pid:190427) [services] - registered service "diagnostic-languageserver"
2020-12-18T14:01:26.966 INFO (pid:190427) [services] - registered service "eslint"
2020-12-18T14:01:26.966 INFO (pid:190427) [services] - eslint langserver state change: stopped => starting
2020-12-18T14:01:26.985 INFO (pid:190427) [language-client-index] - eslint started with 190474
2020-12-18T14:01:27.195 INFO (pid:190427) [services] - registered service "stylelint"
2020-12-18T14:01:27.196 INFO (pid:190427) [services] - stylelint langserver state change: stopped => starting
2020-12-18T14:01:27.209 INFO (pid:190427) [language-client-index] - stylelint started with 190488
2020-12-18T14:01:27.212 INFO (pid:190427) [plugin] - coc.nvim 0.0.79-5a4d0212bf initialized with node: v12.20.0 after 318ms
2020-12-18T14:01:27.213 INFO (pid:190427) [services] - eslint langserver state change: starting => running
2020-12-18T14:01:27.216 INFO (pid:190427) [services] - service eslint started
2020-12-18T14:01:27.225 INFO (pid:190427) [services] - registered service "tsserver"
2020-12-18T14:01:27.281 INFO (pid:190427) [services] - stylelint langserver state change: starting => running
2020-12-18T14:01:27.282 INFO (pid:190427) [services] - service stylelint started
2020-12-18T14:01:39.424 INFO (pid:190427) [attach] - receive notification: showInfo []
```

**Comment by ghost** [comment]
This works though: https://github.com/bmatcuk/coc-stylelintplus

**Comment by sQVe** [comment]
@kdydo That works better - but lacks support for running autofix via command or format. 😞

**Comment by yusuphgammut** [comment]
I have the same issue and the output of `:CocInfo` is the same as @sQVe with the difference that I'm using iTerm2. Would like to fix it but don't know how to debug that.

**Comment by sQVe** [comment]
> I have the same issue and the output of `:CocInfo` is the same as @sQVe with the difference that I'm using iTerm2. Would like to fix it but don't know how to debug that.

`coc-stylelint-plus` now works fine for my use case.

**Comment by yusuphgammut** [comment]
Yes, worked nice for me too. Thanks!

**Comment by osenvosem** [comment]
`coc-stylelint` doesn't work for me as well. `coc-stylelint-plus` works better.
